code/messaging/src/main/java/com/adobe/marketing/mobile/messaging/MessageAssetDownloader.java

```diff
@@ -126,13 +126,13 @@ private void clearCachedAssetsNotInList(final File cacheAsset, final List<String
     private HashMap<String, String> extractMetadataFromResponse(final HttpConnecting response) {
         final HashMap<String, String> metadata = new HashMap<>();
 
-        final String lastModifiedProp = response.getResponsePropertyValue(MessagingConstants.HTTP_HEADER_LAST_MODIFIED);
-        final Date lastModifiedDate = TimeUtils.parseRFC2822Date(
-                lastModifiedProp, TimeZone.getTimeZone("GMT"), Locale.US);
-        final String lastModifiedMetadata = lastModifiedDate == null
-                ? String.valueOf(new Date(0L).getTime())
-                : String.valueOf(lastModifiedDate.getTime());
-        metadata.put(MessagingConstants.HTTP_HEADER_LAST_MODIFIED, lastModifiedMetadata);
+//        final String lastModifiedProp = response.getResponsePropertyValue(MessagingConstants.HTTP_HEADER_LAST_MODIFIED);
+//        final Date lastModifiedDate = TimeUtils.parseRFC2822Date(
+//                lastModifiedProp, TimeZone.getTimeZone("GMT"), Locale.US);
+//        final String lastModifiedMetadata = lastModifiedDate == null
+//                ? String.valueOf(new Date(0L).getTime())
+//                : String.valueOf(lastModifiedDate.getTime());
+//        metadata.put(MessagingConstants.HTTP_HEADER_LAST_MODIFIED, lastModifiedMetadata);
```

Comment:
aren't these values needed when downloading assets?

Reply:
yes!  the method from `TimeUtils` was removed from core but recently added back in.  i'll uncomment this now that we have the method again